content/_index.html

```diff
@@ -15,8 +15,11 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
 		Play with FSH <i class="fas fa-fish "></i>
 	</a>
+	<a class="btn btn-lg btn-info mr-3 mb-4" target="_blank" href="/courses/">
+		On-Line Course <i class="fas fa-graduation-cap "></i>
```

Comment:
```suggestion
		Online Course <i class="fas fa-graduation-cap "></i>
```

[From what I can tell](https://www.usa.gov/style-guide/style-guidance), `online` is preferred to `on-line`